uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/grid/selections/impl/BaseCellSelectionManager.java

```diff
@@ -243,7 +244,8 @@ public boolean startEditingCell(final Point2D ap) {
         }
 
         return edit(uiRowIndex,
-                    ci);
+                    ci,
+                    Optional.of(ap));
```

Comment:
Does the `ap` stand for absolute position? if yes, I have concern why it is passed in place of argument declared as `relativeLocation`.

Reply:
JavaDocs states ```ap``` is "Canvas coordinate relative to the GridWidget" so ```ap``` is a little misleading. I've changed it to ```rp``` throughout.